torchci/components/benchmark/llms/common.tsx

```diff
@@ -1,6 +1,11 @@
 import { BranchAndCommit } from "lib/types";
 
-export const BENCHMARKS = ["gpt_fast_benchmark"];
+export const REPOS = ["pytorch/pytorch", "pytorch/executorch"];
+export const REPO_TO_BENCHMARKS: { [k: string]: string[] } = {
+  "pytorch/pytorch": ["gpt_fast_benchmark"],
+  "pytorch/executorch": ["android-perf", "apple-perf"],
+};
+export const EXCLUDED_METRICS: string[] = ["load_status"];
```

Comment:
If a model fail to load or run, would the data point show up on the dashboard? 
If no, I guess we can probably hide metric `load_status` from the dashboard because all models must have been loaded successfully to be able to show on the dashboard.